Projects/App/Sources/Generic/AppFlow.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import Presentation
+import Embark
+import Flow
+import hCore
+import UIKit
+
+struct AppFlow {
+    private let rootNavigationController = UINavigationController()
+    
+    let window: UIWindow
+    
+    init(window: UIWindow) {
+        self.window = window
+        self.window.rootViewController = rootNavigationController
+    }
+}
+
+struct OnboardingFlow: Presentable {
+    public func materialize() -> (UIViewController, Disposable) {
+        let (viewController, future) = EmbarkOnboardingFlow().materialize()
+        let bag = DisposeBag()
+        
+        bag += future.onValue({ (redirect) in
+            switch redirect {
+            case .mailingList:
+                break
+            case .offer(let ids):
+                bag += viewController.present(WebOnboarding(webScreen: .webOffer(ids: ids))).onResult { result in
+                    switch result {
+                    case .success:
+                        bag += viewController.present(PostOnboarding())
+                    case .failure:
+                        break
```

Comment:
Should we have some error handling here?

Reply:
Whats the default way of presenting errors? an alert?